actors/miner/src/expiration_queue.rs

```diff
@@ -35,8 +36,11 @@ const ENTRY_SECTORS_MAX: u64 = 10_000;
 pub struct ExpirationSet {
     /// Sectors expiring "on time" at the end of their committed life
     pub on_time_sectors: BitField,
-    /// Sectors expiring "early" due to being faulty for too long
+    /// Sectors expiring "early", due to proof expiration, difference from on_time_sectors is that
+    /// the pledge for them is not counted in the ExpirationSet as it is manually assessed during expiration.
     pub early_sectors: BitField,
+    /// Sectors expiring "early" due to being faulty for too long
+    pub faulty_sectors: BitField,
```

Comment:
Can we choose a new name for `early_sectors`? It's strange that the old `early_sectors` are now `faulty_sectors`, and now a different type of expiring sectors are `early_sectors`. I wouldn't mind using `early_sectors` as a term for the union of these two sets.

Reply:
Yeah, I will look into it.